kernel/util/mlog-base/util/Logger.hh

```diff
@@ -99,31 +99,31 @@ void Logger<Filter>::write(ARGS&&... args) const {
 }
 
 static Logger<FilterAny> testLog("Test");
-inline static void test_log(const char *file_line, const char *msg, const char *expr) {
-  if (testLog.isActive(mlog::TextError::VERBOSITY)) {
-    testLog.write<mlog::TextError>(file_line, msg, expr);
-  }
-}
+#define MLOG_TEST(logger, expr) do { \
+    if (expr) \
+      logger.write<mlog::TextSuccess>(PATH, "Success:", TOSTRING(expr)); \
```

Comment:
because `logger` is a macro argument that is expanded textually, we should use `(logger).` instead of `logger.`